### ISSUE TITLE
[Feature Request] Delete manga from webui

### ISSUE THREAD

**Issue by Konosprod** [hkalexling/Mango#279]
**Is your feature request related to a problem? Please describe.**
I know that mango reflects what's on the library directory, but when you are using it remotly, you have to log via SSH, to delete an entry, which can be a bit tiring. Also, if you have the ability within the application to get content, via plugins, you should have the abitlity to remove it.

**Describe the solution you'd like**
Having a button somewhere as an admin to delete an entry. It could be on a chapter, or a top  level entry.

**Describe a small use-case for this feature request**
If you have finished to read a chapter, or a manga, and you have a limited space, or you just don't want to keep it, you could delete the entry.

I read the #263 which was a bit like my request, but the solution was too complicated.

Thank you again for your hardwork and your time !

**Comment by dbx10** [comment]
This is a necessary feature

**Comment by tr7zw** [comment]
This would be really helpful to clean some stuff up, without having to go through the SSH and then trying to find the correct file(s).

**Comment by Leeingnyo** [comment]
Yes. before this, I think, a user permission system should be firstly implemented. I'll create an issue for that one soon (in this week).

**Comment by tr7zw** [comment]
Not sure if this has to depend on implementing a permission system first. For now, it could be just enabled for admin accounts.
